developer-rchain-coop/src/views/token-swap.hbs

```diff
@@ -16,49 +16,82 @@
+                <div class="body">
+                    <p>We no longer use this term in favor of REV issuance that more accurately describes the process. The term “token swap” was introduced before a process had been established and it stuck within the community.</p>
+                </div>
                 <div class="title">My RHOC is stored in a multi-signature wallet, is this supported at the launch?</div>
-                <div class="body">No. Multi-signature wallets are not supported. The issuance of REV requires you to control your Ethereum private keys to receive your REV. To receive REV for RHOC you must have your RHOC in an Ethereum account where you control the private keys before the finalization of RHOC balances just prior to the launch of the RChain blockchain. We recommend you do this before the snapshot.</div>
-                <div class="title">My RHOC is stored in an exchange, is this supported at the launch?</div>
-                <div class="body">Maybe. However neither the RChain Cooperative nor the development team can make that guarantee. The RChain Cooperative has contacted exchanges listing RHOC, requesting that they halt trade at the correct time and and credit users with REV once the network launches. If you want to assure a successful receipt of REV for RHOC, we strongly recommend moving your RHOC out of an exchange and into an Ethereum account where you control the private keys.</div>
-                <div class="title">My RHOC is stored in a hardware wallet, is this supported at launch?</div>
-                <div class="body">As long as you control the private key, this conforms to the criteria mentioned above.</div>
+                <div class="body">
+                    <p>No. Multi-signature wallets are not supported. The issuance of REV requires you to control your Ethereum private keys to have received your REV. To receive REV for RHOC you must have had your RHOC in an Ethereum account where you control the private keys before the finalization of RHOC balances, which occured on Jan 28, 2020, just prior to the launch of the RChain blockchain on Feb 25, 2020.</p>
+                </div>
+                <div class="title">My RHOC was stored in an exchange, is this supported at the launch?</div>
+                <div class="body">
+                    <p>If the exchange was Kucoin and you didn't withdraw your RHOC prior to May 8 2020 Kucoin will have seized your REV.</p>
```

Comment:
Kucoin's position is not something we want to take on as our own. "KuCoin claims X" is as far as I would go.

Reply:
agree